index.html

```diff
@@ -0,0 +1,110 @@
+                    </div>
+                </nav>
+            </section>
+        </header>
+            <div class="top"></div>
+            <div class="flex w-100 h-100 hero-container">
+                <div class="flex txt-center quiz-heading">
+                    <p class="txt-lg lt-bold">Are you even a true <span class="txt-xl highlight-txt" >sitcom</span> fan?</p>
+                    <p class="txt-lg">Let's see now!</p>
+                    <button class="btn btn-sm see-quiz-btn">See Quizzes</button>
+                </div>
+                <div class="hero-img-container">
+                    <div class="img-container">
+                        <img src="./assets/hero.svg" alt="">
+                    </div>
+                </div>
+
+            </div>
+            <section class="flex-container quiz-card-container p-1">
+                <div class="vertical-card overflow-hidden">
+                    <div class="card-header">
+                    <div class="img-container">
+                        <img class="w-100 h-100" src="https://vistapointe.net/images/the-office-us-4.jpg" alt="">
+                    </div>
+                </div>
+                <div class="card-content p-1">
+                    <div class="card-title txt-md lt-bold">the office</div>
+                    <div class="card-description txt-sm">
+                        Would you rather be feared or loved?
+                    </div>
+                    <div class="card-footer">
+                        <div class="w-100 card-btn-container">
+                            <button class="btn btn-sm cta-btn w-100 quiz-cta-btn">Play</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="vertical-card overflow-hidden">
+                <div class="card-header">
+                    <div class="img-container">
+                        <img class="w-100 h-100" src="https://images.indianexpress.com/2020/05/how-i-met-your-mother-759.jpg" alt="">
+                    </div>
+                </div>
+                <div class="card-content p-1">
+                    <div class="card-title txt-md lt-bold">How I Met Your Mother</div>
+                    <div class="card-description txt-sm">
+                        Whenever I'm sad, I stop being sad and be awesome instead.
+                    </div>
+                    <div class="card-footer">
+                        <div class="w-100 card-btn-container">
+                            <button class="btn btn-sm cta-btn w-100 quiz-cta-btn">Play</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <footer>
+            <section class="footer-container">
+                <div class="flex author-intro">
+                    <p class="txt-md" >
+                        Made by <span class="special-txt txt-lg" >SHEKHAR</span>
```

Comment:
This thing can be done just by p tag in the footer. Please do not use unnecessary divs and section tags.

Reply:
I understood, but the section is because I'll add my social links and some information in footer in future.

Reply:
Removed the div